protostar/commands/init/project_creator/new_project_creator.py

```diff
@@ -120,8 +120,9 @@ def _new_project_config(
 
     @staticmethod
     def _validate_project_name(name: str):
-        pattern = r"^[a-zA-Z0-9][a-zA-Z0-9_]*$"
+        # https://github.com/software-mansion/scarb/blob/main/scarb/src/core/package/name.rs#LL42C9
+        pattern = r"^(?!_$)[a-zA-Z_][0-9a-zA-Z_]*$"
```

Comment:
So is `__` (two underscores) is a valid package name?

Reply:
Yes, it is